build/azure-pipelines/darwin/sql-product-build-darwin.yml

```diff
@@ -163,10 +164,10 @@ steps:
   #   condition: and(succeeded(), eq(variables['RUN_TESTS'], 'true'))
 
   - task: DownloadBuildArtifacts@0
-    displayName: 'Download arm64 and x64 packages'
+    displayName: "Download arm64 and x64 packages"
```

Comment:
hmm, I recently enabled "Format on save" but not sure I like all the random edits every time I save a file.